scripts/us_eia/open_data/generate_jsonl_for_bq.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Converts a single dataset into 2 jsonl's: one each for series and category data.
+
+These jsonl's can then be imported to bq using the associated schema bq_schema_*.json
```

Comment:
Might be good to have a README summarizing the sequence and the commands below?

Reply:
added